src/lib/objects/lookup.ts

```diff
@@ -2,12 +2,18 @@ import type { WakuObjectSvelteDescriptor } from '.'
 import { getExternalDescriptor } from './external'
 import { helloWorldDescriptor } from './hello-world'
 import { payggyDescriptor } from './payggy'
+import sandboxLogo from '@waku-objects/sandbox-example/object/logo.svg'
+import sandboxMeta from '@waku-objects/sandbox-example/object/metadata.json'
```

Comment:
Yes, as we discussed, that was my original approach too, but that would have made the lookup logic async and that would in turn require a svelte store for the UI to work correctly. At the end I choose this because it is simpler and it works well for now. Later we can change this to async to support externally hosted objects.